dev/reformat

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
```

Comment:
I think we should rename the script: `reformat-pandas-on-spark`

Reply:
Also updated the PR description, thanks!